core/src/test/resources/spark-events-profiling/spark_rapids_build_info_eventlog

```diff
@@ -0,0 +1,304 @@
+{"Event":"SparkListenerLogStart","Spark Version":"3.1.3"}
```

Comment:
I'd really rather not check in more static files. Can we generate in code similar to:
https://github.com/NVIDIA/spark-rapids-tools/blob/dev/core/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala#L211

Reply:
Thanks @tgravescs! Updated the test cases.

Reply:
thanks for moving the results and comparison, spark_rapids_build_info_eventlog as well if its possible.  Can we just have a smallish number of events in the test case and write them to a file in the code rather then checking in this hardcoded version?

Reply:
removed spark_rapids_build_info_eventlog as well, thanks @tgravescs!